<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="txtLocalIP"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btnGetLocalIP">
     <property name="text">
      <string>获取本机IP</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="txtWebIP"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnGetWebIP">
     <property name="text">
      <string>获取外网IP</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="txtCheckWeb"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnCheckWeb">
     <property name="text">
      <string>检测是否通外网</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
